src/lists/list-item.jsx

```diff
@@ -665,6 +665,7 @@ const ListItem = React.createClass({
               onTouchTap={primaryTogglesNestedList ? this._handleNestedListToggle : onTouchTap}
               ref="enhancedButton"
               style={Object.assign({}, styles.root, style)}
+              touchRippleOpacity={0.1}
```

Comment:
I believe that we could use `0.1` as the default value instead of `0.16`.
As far as I have looked at the code.
This is the only place where the default value is used.
So, what about removing this line and changing the default value?